CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # Okta Python SDK Changelog
 
+## v1.7.0
+- Update SDK according to openapi spec v2.4.0.
```

Comment:
Can we please add more details about the relevant updates? i.e new endpoints/models, etc?

Reply:
Added